studio/components/interfaces/Settings/ProjectUsageBars/ProjectUsageBars.tsx

```diff
@@ -216,8 +236,18 @@ const ProjectUsage: FC<Props> = ({ projectRef }) => {
                                 </td>
                               </>
                             )}
-                          </tr>
-                        )
+                          </tr>,
+                          featureFootnotes[feature.key] && (
+                            <tr>
```

Comment:
Does this `<tr>` need a key?

Reply:
yeah i think they do since we're returning an array of elements @kevcodez

Reply:
What would be an appropriate key? The other tr has `key={feature.title}` so maybe just `key={feature.title}-footnote`? to keep it dynamic and not have duplicate keys?

Reply:
Added key